theseus/labs/lie/functional/so3_impl.py

```diff
@@ -1019,11 +1071,17 @@ def _skew_symm(matrix: torch.Tensor) -> torch.Tensor:
     F = F.pow(-1)
 
     u_term: torch.Tensor = u @ (F * _skew_symm(ut @ grad_u))
-    u_term = torch.einsum("n...ij, n...j->n...ij", u_term, s)
+    # u_term = torch.einsum("n...ij, n...j->n...ij", u_term, s)
```

Comment:
Delete comment?

Reply:
I would like to keep this to understand what the following lines do.

Reply:
In that case, can you write something like `"The next X lines below are equivalent to..."`. And you can also include why this is better than using einsum directly (is it because of `torch.autograd.jacobian` with `vectorize=True` compatibility in torch > 2.0?)

The current version of this comment will not be very meaningful to other people reading this code.

Reply:
Added.